Skip explicit read back for state control properties (#150)

tensoid opened the pull request:
Updates the writeProperty() method in ManagementClient to skip read-back validation for property IDs 5 (PID_LOAD_STATE_CONTROL) and 6 (PID_RUN_STATE_CONTROL). These properties return a resulting state code, which may differ from the written value, causing false negatives during validation.

Fixes #150

bmalinowsky reviewed (changes requested):
I think this should be noted in the JavaDoc of `ManagementClient::writeProperty` (line 570), also because the KNX spec itself doesn't mention this, and there is no way this would succeed otherwise.

For example, inserting your adjusted phrase "These properties return a resulting state code, which may differ from the written value" -> "Reading back the property value shall be skipped for properties returning a resulting state code which may differ from the written value".

Maybe insert that is done for PDT_CONTROL.

tensoid commented:
Thanks for the feedback. I updated the JavaDoc and replaced the values with PropertyAccess.PID constants. Let me know if anything else needs changing.

bmalinowsky commented:
@tensoid Thanks!
